state-chain/custom-rpc/src/lib.rs

```diff
@@ -467,8 +483,9 @@ pub trait CustomApi {
 	fn cf_max_swap_amount(&self, asset: RpcAsset) -> RpcResult<Option<AssetAmount>>;
 	#[subscription(name = "subscribe_pool_price", item = PoolPrice)]
 	fn cf_subscribe_pool_price(&self, from_asset: RpcAsset, to_asset: RpcAsset);
-
-	#[subscription(name = "subscribe_prewitness_swaps", item = Vec<AssetAmount>)]
+	#[subscription(name = "subscribe_pool_price_v2", item = SubscriptionUpdate<PoolPriceV2>)]
+	fn cf_subscribe_pool_price_v2(&self, from_asset: RpcAsset, to_asset: RpcAsset);
```

Comment:
Also for this new subscription can we use three arguments (base_asset, quote_asset, and side). We had problems with people not understanding the price for side is different.

Reply:
Sorry, can we also introduce a cf_pool_price_v2, that matches the new subscription arguments and response structure.

Reply:
Added cf_pool_price_v2 call and PoolPriceV2, which contains
```
buy_price: PoolPriceV1,
sell_price: PoolPriceV1,
```